### PR TITLE
Set price expander in Cluster Autoscaler for GCE

### PR DESCRIPTION
**Pull request by mwielgus**
With CA 0.6 we will make price-preferred node expander the default one for GCE. For other cloud providers we will stick to the default one (random) until the community implement the required interfaces in CA repo.

https://github.com/kubernetes/autoscaler/issues/82

cc: @MaciekPytel @aleksandra-malinowska

### CODE REVIEW

File: cluster/gce/gci/configure-helper.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+qq  #!/bin/bash
```
Comment (MaciekPytel) [inline cluster/gce/gci/configure-helper.sh]: Is that a typo? I have no idea what it's supposed to do and it breaks the tests.

**Comment by MaciekPytel** [comment]
@mwielgus I wonder if we want to add a custom env variable for everything? Maybe you should just add something like AUTOSCALER_FLAGS or AUTOSCALER_CUSTOM_OPTIONS and put any flags required in there? I'm thinking whether to add `--balance-similar-node-groups=true` as a default at some point. And we're likely to have even more things we may want to turn on in future.

A nice side-benefit would be allowing users to set all the flags we provide in cluster-autoscaler (like scale-down delays) when using kube-up.sh, etc. 
The downside would be, of course, changing it would override all our default values not just a single one.

**Comment by mwielgus** [comment]
Discussed offline, posting the summary here.
In general we want a separate env with every non-trivial configuration element. Otherwise we would end up with just couple envs SCHEDULER_FLAGS, KUBELET_FLAGS, CA_FLAGS which would be unparseable and super implementation-specific.

**Comment by grodrigues3** [comment]
@mwielgus is there an associated issue for this? Please reference it in the PR body.

**Comment by mwielgus** [comment]
ref: https://github.com/kubernetes/autoscaler/issues/82

**Comment by dchen1107** [comment]
@k8s-bot pull-kubernetes-kubemark-e2e-gce test this

**Pull request merged**